### ISSUE TITLE
stylelint doesn't flag quoted css values as errors

### ISSUE THREAD

**Issue by broomstick73** [stylelint/stylelint#4208]
> Clearly describe the bug
the following css is invalid but not flagged by stylelint
.class { color: "blue"; }

> Which rule, if any, is the bug related to?
there doesn't seem to be a rule for this? maybe string-quotes?

> What CSS is needed to reproduce the bug?
```
css
a { 
color: "blue";
 }
```

> What stylelint configuration is needed to reproduce the bug?
seems to be any configuration

> Which version of stylelint are you using?
10.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?
VS Code plugin

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?
no; basic css

> What did you expect to happen?
expected a warning to be flagged

> What actually happened (e.g. what warnings or errors did you get)?
no warning was flagged; css is invalid and not applied; browser dev tools show it as invalid thus not applied.

**Comment by hudochenkov** [comment]
Stylelint doesn't check validity. Use [`stylelint-csstree-validator` plugin](https://github.com/csstree/stylelint-validator) for this.
